appium/webdriver/extensions/android/network.py

```diff
@@ -34,15 +38,15 @@ class NetSpeed(object):
 class Network(webdriver.Remote):
 
     @property
-    def network_connection(self):
+    def network_connection(self) -> int:
         """Returns an integer bitmask specifying the network connection type.
 
         Android only.
         Possible values are available through the enumeration `appium.webdriver.ConnectionType`
         """
         return self.execute(Command.GET_NETWORK_CONNECTION, {})['value']
 
-    def set_network_connection(self, connection_type):
+    def set_network_connection(self, connection_type: int) -> Any:  # TODO Check return type
```

Comment:
Perhaps None ( or `''`)

Reply:
It returns the connection state after the change: https://github.com/appium/appium-android-driver/blob/64cd063862e7126c59a9f28d101250c08e3f1858/lib/commands/network.js#L91

Reply:
Thanks, it seems `the connection state` as return val also from my checking actual behavior.